src/test/java/hw5/pages/UserTable.java

```diff
@@ -0,0 +1,117 @@
+package hw5.pages;
+
+import hw5.pages.components.HeaderMenu;
+import hw5.pages.components.LeftSideMenu;
+import hw5.pages.components.RightSideBar;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserTable extends BaseElements {
+    private HeaderMenu headerMenu;
+    private LeftSideMenu leftSideMenu;
+    private RightSideBar rightSideBar;
+    private WebDriver driver;
+
```

Comment:
Why do you need these fields in this class?

Reply:
moved to BaseElements and remove driver